app/Invoice.php

```diff
@@ -9,6 +9,10 @@ class Invoice extends Model
 {
     use HasFactory;
 
+    public $casts = [
+        'time_issued' => 'datetime'
```

Comment:
We may not even need this, but this will allow you to avoid constantly parsing this data in your controllers/traits that look at this data.